pkg/reconciler/mongodb/resources/labels.go

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
this one should be part of the follow up PR when we start adding the receive adapter bits. 
I think we can remove it for now